src/app/campaign-list/card/campaign-card.component.scss

```diff
@@ -21,23 +23,23 @@ h3 {
 }
 .status {
   padding: 2px 4px;
-  color: #000;
+  color: map-get($prx-gray, 900);
```

Comment:
making this off black rather than true black makes the contrast issue we have on these cards worse. True black had a better contrast ratio than white with all these colors, but it still wasn't a11y 4.5:1 contrast compliant.

Reply:
Ah, this is a good use of the palette's `<hue>-contrast`, so I've switched over to that. It _looks_ like the `@angular/material/_theming.scss` file is just mixin, function and variable definitions, so it should be no problem to import that wherever we need to use those function calls (as far as I can tell).